content/en/docs/examples/microservices-istio/setup-kubernetes-cluster/index.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Setup a Kubernetes Cluster
+overview: Set up your Kubernetes cluster for the tutorial.
+weight: 2
+---
+
+{{< boilerplate work-in-progress >}}
+
+In this module, you set up a Kubernetes cluster that has Istio installed and a
+namespace to use throughout the tutorial.
+
+{{< warning >}}
+If you are in a workshop and the instructors provide a cluster for you,
+go to [setting up your local computer](/docs/examples/microservices-istio/setup-local-computer).
+{{</ warning >}}
+
+1.  Ensure you have access to a [Kubernetes cluster](https://kubernetes.io/docs/tutorials/kubernetes-basics/).
+    You can use the [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/quickstart) or the
+    [IBM Cloud Kubernetes Service](https://cloud.ibm.com/docs/containers?topic=containers-getting-started).
+
+1.  Connect to your cluster and create an environment variable to store the name
+    of a namespace that you will use when you run the tutorial commands.
+    You can use any name, for example `tutorial`.
+
+    {{< text bash >}}
+    $ export NAMESPACE=tutorial
+    {{< /text >}}
+
+1.  Create the namespace:
+
+    {{< text bash >}}
+    $ kubectl create namespace $NAMESPACE
+    {{< /text >}}
+
+    {{< tip >}}
+    If you are an instructor, you should allocate a separate namespace per each
+    participant. The tutorial supports work in multiple namespaces
+    simultaneously by multiple participants.
+    {{< /tip >}}
+
+1.  Install Istio with strict mutual TLS enabled and select the `strict mutual
```

Comment:
"select the strict mutual TLS tab" is part of "Install Istio with strict mutual TLS". So it is not ***and*** between the two parts of the sentence, but rather the second part is a step in the first part. So it should be something like: install Istio with strict mutual TLS enable by following the steps at ... and selecting the "strict mutual TLS tab".